tests/scripts/test_scripts.py

```diff
@@ -413,13 +414,21 @@ def test_train_rl_sac(tmpdir):
     assert isinstance(run.result, dict)
 
 
-EVAL_POLICY_CONFIGS = [
-    {"videos": True},
-    {"videos": True, "video_kwargs": {"single_video": False}},
+# check if platform is macos
+
+EVAL_POLICY_CONFIGS: List[Dict] = [
     {"reward_type": "zero", "reward_path": "foobar"},
     {"rollout_save_path": "{log_dir}/rollouts.pkl"},
 ]
 
+if platform.system() != "Darwin":
```

Comment:
Slight preference for `sys.platform` over `platform.system()` -- on Linux `platform.system()` runs uname and returns fine-grained information which is a bit overkill here. But doesn't really matter: will be same as sys.platform on non-Linux systems anyway I think. https://stackoverflow.com/questions/4553129/when-to-use-os-name-sys-platform-or-platform-system